tests.py

```diff
@@ -73,9 +74,9 @@ def testParseArguments(self):
     options = de_test_object.ParseArguments()
     usage_string = de_test_object._argument_parser.format_usage()
     expected_usage = (
-        'usage: {0} [-h] [-r DOCKER_DIRECTORY] '
-        '{{mount,list,history}} ...\n'.format(os.path.basename(__file__)))
-    self.assertEqual(expected_usage, usage_string)
+        r'usage: (setup.py|{0}) \[-h\] \[-r DOCKER_DIRECTORY\] '
+        '{{mount,list,history}} ...\n').format(os.path.basename(__file__))
+    self.assertRegexpMatches(usage_string, expected_usage)
```

Comment:
This seems to be deprecated in Python 3 https://docs.python.org/3/library/unittest.html, if you're planning to support Python 3 maybe add an issue for this.

Reply:
Added to  #7